solutions/cses-1619.mdx

```diff
@@ -260,53 +249,55 @@ Coordinate compress interval endpoints and only compute the prefix sum array for
 interesting intervals.
 
 <LanguageSection>
-
 <CPPSection>
 
 ```cpp
 #include <bits/stdc++.h>
 using namespace std;
 
 int main() {
-	ios_base::sync_with_stdio(0);
-	cin.tie(0);
-
-	int N;
-	cin >> N;
-	vector<pair<int, int>> customers;
-	for (int i = 0; i < N; ++i) {
-		int a, b; cin >> a >> b;
-		customers.push_back({a, 1});
-		customers.push_back({b + 1, -1});
+	int n;
+	cin >> n;
+	vector<pair<int, int>> times;
+	for (int i = 0; i < n; i++) {
+		int start, end;
+		cin >> start >> end;
+		times.push_back({start, 1});
+		times.push_back({end + 1, -1});
 	}
 
-	sort(customers.begin(), customers.end());
+	sort(times.begin(), times.end());
 
 	int curr = 0;
-	int l = 0;
-	vector<long long> pfx(2*N + 1); // prefix sum array
-	vector<int> arr(2*N);
-
-	for (int i = 0; i < 2*N; ++i) { // coordinate compression
-		if(i == 0) { customers[i].first = 0; }
-		else if(customers[i].first > curr) { l++; curr = customers[i].first; } // we move a pointer
-		arr[l+1] += customers[i].second;
-		// pfx[l+1] += customers[i].second; implementation without arr, merges all interval end and starts here
+	int at = 0;
+	// Stores how much the # of people in the restaurant changes each time step
```

Comment:
still kinda weird wording

Reply:
`compress` is definitely not a better name

Reply:
its more straight forward

Reply:
is it the difference in the numbers of customers?

Reply:
i guess?

Reply:
better off making the comment that then lol

Reply:
it's the same thing as the comment

Reply:
difference across the numbers of customers?? i think its better wording than the current version..

Reply:
at this point i feel like it's your word against mine
@ryanchou-dev